src/components/PostPageDetail/index.js

```diff
@@ -668,7 +666,6 @@ const PostPageDetailIdComponent = (props) => {
             onChangeText={(value) => setTextComment(value)}
             onPress={onComment}
             loadingPost={loadingPost}
-            isKeyboardOpen={isKeyboardOpen}
           />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isKeyboardOpen` prop is removed from the `WriteComment` component. As mentioned earlier, ensure that this prop is not used in the `WriteComment` component.
```diff
- isKeyboardOpen={isKeyboardOpen}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->